cvmfs/cvmfs_server

```diff
@@ -1965,17 +1977,18 @@ CVMFS_AUTO_GC=true
 EOF
   fi
 
-  # make sure that the config file does not exist, yet
-  remove_apache_config_file "cvmfs.${name}.conf" || true
+  if [ $configure_apache -eq 1 ]; then
+    # make sure that the config file does not exist, yet
+    remove_apache_config_file "cvmfs.${name}.conf" || true
```

Comment:
unrelated to this diff, but it might be preferable to rename existing config files rather than to delete them.